src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private Fruit fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, Fruit fruit, int quantity) {
+        this.operation = operation;

```

Comment:
In the constructor, add validation logic for the 'fruit' and 'operation' fields to check if they are not null.

Reply:
ok, done